tasks/sync-granule/GranuleFetcher.js

```diff
@@ -158,6 +158,7 @@ class GranuleFetcher {
         granuleId: granule.granuleId,
         dataType: collectionName,
         version: collectionVersion,
+        provider: this.provider.id,
```

Comment:
Just as an FYI, I updated `sync-granule` because for the ingestGranule workflow the granules output collection didn't contain the provider for the lzards-backup task to use as metadata resulting in failures there. The `provider` was available to the `GranuleFetcher` and seemed like a logical place to include it.